content/en/apps/guides/performance/purging.md

```diff
@@ -105,7 +105,9 @@ This function takes four parameters:
 - `userCtx`, an object with the user's `roles` as fields. For more information read the [documentation for the User Context Object](https://docs.couchdb.org/en/stable/json-structure.html#userctx-object).
 - `contact`, the contact document of a patient or other contact who has reports about them.
 - `reports`, an array of all reports for that subject that are present on the server.
-- `messages`, an array of sms messages that the contact has sent or received
+- `messages`, an array of sms messages that the contact has sent or received.
```

Comment:
Line 103 above says "this function takes four parameters" which is now out of date.